application/views/admin/dataentry/vvimport.php

```diff
@@ -158,18 +158,3 @@
         <?php } ?>
 
 </div></div></div>
-<?php
-App()->getClientScript()->registerScript('VVImportBSSwitcher', "
-LS.renderBootstrapSwitch();
-
-$('#noid').on('switchChange.bootstrapSwitch', function(event, state) {
```

Comment:
Is this correct, to remove logic like this without replacing it with something?
What's the proper test for this piece of code?

Reply:
Added this functionality.

Reply:
Great, I will unresolve this so that Tibur can find it later.